kasa/tests/fixtures/smart/L920-5(EU)_1.0_1.1.3.json

```diff
@@ -122,7 +122,7 @@
         "factory_default": false,
         "ip": "127.0.0.123",
         "is_support_iot_cloud": true,
-        "mac": "1C-61-B4-00-00-00",
+        "mac": "B4-B0-24-00-00-00",
```

Comment:
Perhaps we should add the fixture using a different name to avoid losing information about macs?